components/Features/Features.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["Features.scss"],"names":[],"mappings":"AAAA;EACE","file":"Features.css"}
```

Comment:
What are these map files?  Is it an IDE thing?  Maybe we can add a `.gitignore` entry.